src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Admin/Controllers/RequestsController.cs

```diff
@@ -29,7 +33,8 @@ public RequestsController(IFeedbackService feedbackService)
         /// <returns>Action result</returns>
         public ActionResult Index()
         {
-            var feedbacks = _feedbackService.Get().ConvertAll(f => new RequestsViewModel(f));
+            var feedbacks = _feedbackService.Get().Select(f =>new RequestsViewModel(f)).ToList();
+            //var feedbacks = ((List<Feedback>)_feedbackService.Get()).ConvertAll(f => new RequestsViewModel(f));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS125)